src/main/kotlin/com/jay/racingcar/view/OutputView.kt

```diff
@@ -17,11 +20,21 @@ object OutputView {
         println("실행 결과")
     }
 
+    private fun printWinnerMessage() {
+        println("가 최종 우승했습니다.")
+    }
+
     fun printResult(racingCars: RacingCars) {
         racingCars.racingCars
             .forEach {
-                println(DISTANCE_MARK.repeat(it.position))
+                print(it.getNameValue() + CAR_NAME_MARK)
+                println(DISTANCE_MARK.repeat(it.getPosition()))
             }
         println()
     }
+
+    fun printWinners(winners: List<RacingCar>) {
+        print(winners.stream().map { it.getNameValue() }.toList().joinToString(separator = ","))
```

Comment:
```suggestion
        print(winners.joinToString(separator = ",") { it.getNameValue() })
```
코틀린에서는 자바와 다르게, stream으로 변환해주지 않아도 됩니다.
또한, 컬렉션을 다른 것으로 변환해주는 함수들 (filter, map 등)은 실행이 완료되면 매번 새로운 컬렉션을 만들어줍니다.
그래서 map 함수를 호출한 뒤에 toList함수를 호출해주실 필요가 없습니다.
또한, joinToString 함수를 잘 활용하시면 위와 같이 간단하게 처리할 수 있습니다. 😄